llvm/lib/Transforms/Scalar/ConstraintElimination.cpp

```diff
@@ -995,6 +995,11 @@ void State::addInfoFor(BasicBlock &BB) {
       continue;
     }
 
+    if (match(&I, m_Intrinsic<Intrinsic::abs>())) {
```

Comment:
Now that we support a number of intrinsics, we should probably covert that to a switch on the intrinsic ID, so it is easy to see what intrinsics are handled at a glance, as well as making it obvious that they are all handled in the same way